### DISCUSSION TITLE
Automatic upgrade of models

### DISCUSSION THREAD

**Discussion by mayankjobanputra** [huggingface/safetensors#203] [Q&A] [answered]
I see PRs on my models. Just wanted to make sure it's a legit PR from HuggingFace and not any kind of malicious PR.

![image](https://user-images.githubusercontent.com/10355927/227440768-735acecc-6ff5-4c87-a39a-c0068afc9283.png)

**Comment by Narsil** [comment, marked as answer]
Indeed it's coming from a huggingface bot. Actually it's the API running the widget of this model that initiates that PR so that it can use the safer weights (and slightly faster load for now, potentially much faster load in the future)
